src/containers/TriggerTemplate/triggertemplate.scss

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+@import '~carbon-components/scss/globals/scss/vars';
+
+.bx--data-table-container {
+  overflow: hidden;
+}
+
+.textarea {
+  display: block;
+  overflow: hidden;
+  resize: both;
+  overflow: auto;
+  padding: 10px;
+  width: 100%;
+  font-family: ibm-plex-mono, monospace;
+  box-shadow: 1px 1px 4px rgba(black, .2);
+}
+
+.triggertemplatetext {
+  width: 90%;
+  text-shadow: 0.1px 0.1px 0 black;
+}
+
+.triggertemplatetext p {
+  font-size: 0.8rem;
+}
+
+.tableComponent {
+  .bx--data-table-container {
+    .bx--table-toolbar {
+      display: none !important;
+    }
+  }
+}
```

Comment:
can you post another pic to see how it rendered with the css?

Reply:
![image](https://user-images.githubusercontent.com/12814972/69529020-50cece00-0f67-11ea-91f2-3ba917dc27b8.png)